pkg/autodiscovery/autoconfig.go

```diff
@@ -102,6 +106,12 @@ func (ac *AutoConfig) serviceListening() {
 	tagFreshnessTicker := time.NewTicker(15 * time.Second) // we can miss tags for one run
 	defer tagFreshnessTicker.Stop()
 
+	// this must be greater than or equal to the poll interval of every config
+	// poller, as a service is considered bare when this ticker has ticked
+	// _twice_ with no associated templates.
+	bareServiceTicker := time.NewTicker(15 * time.Second)
```

Comment:
Any situation where a service could spawn it's configuration later than 30s after starting ? Main question is: should we make this configurable ?

Reply:
I have no idea.  I assume that this approach will be roundly rejected due to this issue (and, meaning that containers don't start logging for 30 seconds) -- but I need advice on a better way :)